Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -82,6 +88,20 @@ public DriveSubsystem(WPIFactory factory, XPropertyManager propManager, ShiftSub
         previousRightTicks = rightDrive.getPosition();
         leftInchesTraveled = 0;
         rightInchesTraveled = 0;
+        
+        drivingStateLatch.addObserver((Observable o, Object arg) -> {
+            if(arg instanceof EdgeType) {
+                EdgeType edge = (EdgeType)arg;
+                String type = "";
+                if(edge == EdgeType.RisingEdge) {
+                    type = "starting";
+                }
+                else if(edge == EdgeType.FallingEdge) {
+                    type = "stopping";
+                }
+                log.info(String.format("Drive %s left: %.2f, right: %.2f", type, this.leftDrive.get(), this.rightDrive.get()));
```

Comment:
I'm not sure what you mean. What is the value that printing the two powers in this location gives us? I could see a use case for periodic logging of power targets, but it seems to me that the numbers printed here will be of little use -- they'll distract more than they enlighten.